github-dark.user.css

```diff
@@ -4639,5 +4639,23 @@
   /*[[syntax-theme]]*/
   /*[[syntax-codemirror]]*/
   /*[[syntax-jupyter]]*/
-
 }
+@-moz-document domain("zuora.com") {
```

Comment:
Maybe add a small comment in `github-dark.css` why this domain is styled. It's not obvious to me.

Reply:
The url used in these iframes contains `github.com` which means we can change this to a regex so it's more specific:
```regex
https://www\.zuora\.com.*github\.com.*
```